packages/bpk-component-accordion/src/BpkAccordionItem.module.scss

```diff
@@ -24,7 +24,13 @@ $bpk-spacing-v2: true;
   &__title {
     height: auto;
 
-    @include bpk-border-bottom-sm($bpk-line-day);
+    &--collapsed {
+      @include bpk-border-bottom-sm($bpk-line-day);
+
+      &-on-dark {
+        @include bpk-border-bottom-sm($bpk-line-night);
```

Comment:
Let's ask design about this (the night token usage)
![Screenshot 2023-03-20 at 13 44 48](https://user-images.githubusercontent.com/89925955/226358410-345227a9-f8f6-454e-86b1-a32e45feea1a.png)

Reply:
Updated the colours.